sections/03-methods.md

```diff
@@ -0,0 +1,65 @@
+
+It is in these moments of outright antagonism that something else emerges: a positive dimension to the conflict, which forces me to get away from the computer and out into the world with my camera, working in "real time" with quotidian material to produce simple videos with few edits or technical conceits. Many of the videos in _Insignificant Surfaces_ were produced as a result of this failure of the programming/video dialectic to synthesize. These videos emerged as the surplus of the disciplinary relation, the "unmarked space" of a programming practice that nonetheless perturbs and irritates the material system of my video art practice into producing something different. If computation is a recurring theoretical concern throughout all my work, its absence is as notable as--and inseparable from--its presence.
+
+## Methods in Practice
+
+So far, I have discussed a theory of interdisciplinary practice and research. In the sections that follow, I will elaborate on a pair of specific research/creation methods that play a significant role within my work, exploring their relationship to the artworks I have created for this project. These methods blend personal strategies that have emerged from the process of creating and theorizing video, sound, and software over the past several years with adaptations of established art and design methods.
+
+In the next section, _Lensing_, I will discuss lensing as a method and apply it to my own work and that of Stan Brakhage. Later, in _Vibrant Bodies_, I will explore the relationship between open source collaborative methods and my software practice via my creative tools _Flocking_ and _Aconite_, and my soundtrack for Izabella Pruska-Oldenhof's film, _Font Màgica_.
+
+## Notes
+
+1. I use the term "design" here expansively to include technical activities such as software architecture and programming. This is in part due to disciplinary affiliations (or lack thereof), in that I do not identify with the fields of computer science or engineering, but more importantly motivated by a long-standing desire to place the emphasis of my technological work on creative and social concerns, not on computation as such
+
+2. Adjacent relations emerge as the incongruous and unexpected byproducts of simultaneity and spatial proximity. Marc Couroux: "Psychedelic adjacencies are generated via the strategic imbrication of overtly incongruent but subliminally (genetically) congenial signals. Such formations are inevitably spawned within a colloidal dispersion in which perpetually recombinant surfaces enter into temporary electrical relationships with one another by virtue of haphazard temporal and spatial proximities." (_Gloassary for a Techno-Sonic Control Society_).
```

Comment:
Couroux' is a remarkably picturesque juxtaposition of vocables for one that, as is typical for this field, flutters at the margins of unintelligibility. The GOOD DOCKTOUR might have referred to perhaps, structures that were "reticulated" or "decussated" : P

cf:
       SQUIRE HEADLONG.
Bravo! Pass the bottle. The very best speech that ever was made.
        MR ESCOT.
It has only the slight disadvantage of being unintelligible.


Gloassary -> Glossary, or, perhaps, Glossolalia